src/main/java/ro/secur/auth/exceptions/ResponseBuilder.java

```diff
@@ -0,0 +1,26 @@
+package ro.secur.auth.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.time.ZonedDateTime;
+
+public class ResponseBuilder<T> extends GlobalException {
```

Comment:
Response Builder is not a Global Exception. Please use inheritance only when you have "IS A" relationship.